ADOPTERS.md

```diff
@@ -5,4 +5,5 @@ This is a likely incomplete list of KubeVirt adopters - end-users and distributo
 | User | Since | Use-Case |
 |-|-|:-|
 | Red Hat | 2016 | As a distributor we are building OpenShift Virtualization on KubeVirt in order to enable VM workloads and -flows on Kubernetes. |
-| [Kubermatic](https://kubermatic.com) | 2019 | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. Follow the [link](https://www.kubermatic.com/products/kubevirt/) for more information. |
\ No newline at end of file
+| [Kubermatic](https://kubermatic.com) | 2019 | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. Follow the [link](https://www.kubermatic.com/products/kubevirt/) for more information. |
+| H3C | 2019 | As a distributor we are running KubeVirt in order to enable VM workloads on Kubernetes and distribute to more and more customer . Follow the  [link](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/) for more product information |
```

Comment:
I think that the former rows are written in cumbersome language. Would you consider writing something like

We distribute KubeVirt as part of [CloudOS](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/)  to enable VM workloads on Kubernetes at customer sites 

I find it simpler shorter and easier to understand.

Reply:
ok,great,let me modify it

Reply:
ok that  haved been  modified